backend/apis/normal-docket-pdf.test.js

```diff
@@ -17,13 +17,15 @@ describe("normal docket pdf file", () => {
     let actual1 = parsedText.charges[1];
 
     expect(actual0.statute).toBe("53-3-202(1)(A)");
-    expect(actual0.description).toBe("NO VALID LICENSE Misdemeanor");
+    expect(actual0.offenseName).toBe("NO VALID LICENSE");
```

Comment:
From https://github.com/JustUtahCoders/utahexpungements.org/blob/ceac1f1e418a35552a3b53f0d9412e0a864cd224/backend/apis/docket-pdf-test-files/normal-docket-pdf.js#L10-L11

This should be `NO VALID LICENSE - EXPIRED`

Reply:
ok yeah that makes sense, I just split description up on the current tests. I should have made my own tests. thanks I will work on the tests